docs/topics/api/reviewers.rst

```diff
@@ -335,14 +335,14 @@ These endpoints allow you to draft comments that can be submitted through the re
 
     :>json int id: The id of the draft comment object.
     :>json string comment: The comment that is being drafted as part of a review. Specific to a line in a file.
-    :>json string filename: The filename a specific comment is related to.
-    :>json int lineno: The line number a specific comment is related to. Please make sure that in case of comments for git diffs, that the `lineno` used here belongs to the file in the version that belongs to `version_id` and not it's parent.
+    :>json string|null filename: The filename a specific comment is related to.
+    :>json int|null lineno: The line number a specific comment is related to. Please make sure that in case of comments for git diffs, that the `lineno` used here belongs to the file in the version that belongs to `version_id` and not it's parent.
     :>json object version: Object holding the :ref:`version <version-detail-object>`.
     :>json int user.id: The id for an author.
     :>json string user.name: The name for an author.
-    :>json string user.url: The link to the profile page for an author.
     :>json string user.username: The username for an author.
-    :>json object canned_response: Object holding the :ref:`canned response <reviewers-canned-response-detail>` if set.
+    :>json string|null user.url: The link to the profile page for an author.
```

Comment:
"The link to the profile page for an author, if the author's profile is public." ?

Reply:
Fixed in 540e0af